src/base.ts

```diff
@@ -47,6 +48,39 @@ export abstract class BitArray {
   public abstract getBit(index: number): boolean;
   public abstract setBit(index: number, value: boolean): void;
   public abstract toBitfield(): Uint8Array;
+
+  public or(bitArray: BitArray): BitArray {
```

Comment:
thoughts on making this return a new BitArray?
If not, I think return type can be `this`

Reply:
It should be this. I'm leaning towards static method that would return new BitArray. Didn't find use case, so not aure if I should add that now?